src/screens/EditExpensesScreen.js

```diff
@@ -1,22 +1,25 @@
-import { SafeAreaView, Text, StyleSheet, TextInput, Pressable, Button } from "react-native";
+import { SafeAreaView, Text, StyleSheet, TextInput, Pressable } from "react-native";
 import React, { useState } from 'react';
 import { db, auth } from "../firebase";
 import { collection, addDoc } from "firebase/firestore";
 import { DatePicker } from "../Components/DatePicker";
+import { CategoryPicker } from "../Components/CategoryPicker";
```

Comment:
perhaps these import statement can be combined :

`import { DatePicker, CategoryPicker } from "../Components"; `

Reply:
I've made the changes!